tests/main/views/test_outcomes.py

```diff
@@ -0,0 +1,918 @@
+                for f_t, variant_params in product((False, True,), _test_most_paths_base_variants)
+            ),
+            (   # and also include some with mixed target-types
+                (
+                    # other_oc_brief_based
+                    False,
+                    # initial_brief_based
+                    True,
+                    # other_oc_data
+                    {
+                        "completed_at": datetime.datetime(2007, 7, 7, 7, 7, 7),
+                        "result": "none-suitable",
+                    },
+                    # initial_data
+                    {
+                        "completed_at": None,
+                        "result": "cancelled",
+                    },
+                    # put_values
+                    {
+                        "completed": True,
+                    },
+                    # expected_status_code
+                    200,
+                    # expected_response_data
+                    {
+                        "outcome": AnySupersetOf({
+                            "completed": True,
+                        }),
+                    },
+                ),
+                (
+                    # other_oc_brief_based
+                    True,
+                    # initial_brief_based
+                    False,
+                    # other_oc_data
+                    {
+                        "completed_at": datetime.datetime(2007, 7, 7, 7, 7, 7),
+                        "awarding_organisation_name": "Lambay Freehold",
+                        "award_value": 54321,
+                        "start_date": datetime.date(2010, 12, 12),
+                        "end_date": datetime.date(2011, 12, 12),
+                    },
+                    # initial_data
+                    {},
+                    # put_values
+                    {
+                        "completed": True,
+                        "award": {
+                            "awardingOrganisationName": "Omphalos",
+                            "awardValue": "00314.1500",
+                            "startDate": "2020-10-10",
+                            "endDate": "2020-11-20",
+                        },
+                    },
+                    # expected_status_code
+                    200,
+                    # expected_response_data
+                    {
+                        "outcome": AnySupersetOf({
+                            "completed": True,
+                            "award": {
+                                "awardingOrganisationName": "Omphalos",
+                                "awardValue": "314.15",
+                                "startDate": "2020-10-10",
+                                "endDate": "2020-11-20",
+                            },
+                        }),
+                    },
+                ),
+            ),
+        )),
+        ids=(lambda val: "EMPTYDCT" if val == {} else None),
+    )
+    def test_most_paths(
```

Comment:
Again, I'd rename to `test_direct_award_and_brief_outcome_scenarios`, so it's clear what's different here to the test in the other module.

Reply:
:+1: